odata-json-format/10 Media Entity.md

```diff
@@ -33,19 +33,24 @@ An individual property or operation response is represented as a JSON
 object.
 
 A single-valued property or operation response that has the
-`null` value does not have a representation; see
-[OData-Protocol](#ODataProtocol).
+`null` value does not have a representation provided that it carries no
+[control information](#ControlInformation) other than
+[`context`](#ControlInformationcontextodatacontext)
+and no [instance annotations](#InstanceAnnotations); see [OData-Protocol](#ODataProtocol).
 
 A property or operation response that is of a primitive type is
-represented as an object with a single name/value pair, whose name is
+represented as an object with a name/value pair whose name is
 `value` and whose value is a [primitive
-value](#PrimitiveValue).
+value](#PrimitiveValue) or `null`.
 
 A property or operation response that is of complex type is represented
-as a [complex value](#ComplexValue).
+as a [complex value](#ComplexValue). If the value is `null`, the JSON object
```

Comment:
Are entity-valued results covered by this paragraph on complex values as well? They are not covered elsewhere.